app/src/main/java/net/programmierecke/radiodroid2/PlayerService.java

```diff
@@ -661,7 +677,11 @@ private void sendMessage(String theTitle, String theMessage, String theTicker) {
                 .setVisibility(NotificationCompat.VISIBILITY_PUBLIC)
                 .setSmallIcon(R.drawable.ic_play_arrow_white_24dp)
                 .setLargeIcon(radioIcon.getBitmap())
-                .addAction(R.drawable.ic_stop_white_24dp, getString(R.string.action_stop), pendingIntentStop);
+                .setStyle(new android.support.v4.media.app.NotificationCompat.MediaStyle()
+                        .setMediaSession(mediaSession.getSessionToken())
+                        .setShowActionsInCompactView(0, 2, 3 /* stop, play/pause, next */))
```

Comment:
1) Maybe we should show play/pause, prev, next - I don't think we need stop there having both skips is more symmetrical.
2) The notification crashes presumable due to these indices, maybe they should be set after ALL actions are added.
```
2019-04-24 21:01:40.409 2541-2541/com.android.systemui E/StatusBar: couldn't inflate view for notification net.programmierecke.radiodroid2/0x1
    java.lang.IndexOutOfBoundsException: Index: 3, Size: 3
        at java.util.ArrayList.get(ArrayList.java:437)
        at android.app.Notification$MediaStyle.makeMediaContentView(Notification.java:6571)
        at android.app.Notification$MediaStyle.makeContentView(Notification.java:6492)
        at android.app.Notification$Builder.createContentView(Notification.java:4323)
        at com.android.systemui.statusbar.notification.NotificationInflater.createContentView(NotificationInflater.java:486)
```

Reply:
OK

Reply:
ad 1) Or do you mean removing the stop button in CompactView, only?

Reply:
swipe to delete sounds good

Reply:
Yes, I mean in compact view where there is maximum 3 buttons

Reply:
It's a bit tricky but I got it working after consulting this here:
https://stackoverflow.com/questions/26496670/dismissing-mediastyle-notifications

However, the "swipe to stop" should probably only be allowed in paused state. This also seems to be the convention. Should I make this a follow-up pull request?

Reply:
I think better change this one.

Reply:
OK

Reply:
The behaviour is pretty much standard, now. Maybe except that we show a stop button instead of a close button.